pkg/client/cache/fifo.go

```diff
@@ -35,6 +35,8 @@ type FIFO struct {
 	// keyFunc is used to make the key used for queued item insertion and retrieval, and
 	// should be deterministic.
 	keyFunc KeyFunc
+	// isOlder is used to accept or reject objects which are older than what's already stored.
+	isOlder IsOlder
```

Comment:
i phrased that badly...  "is used to accept or reject objects which are older than what's stored" isn't accurate.  "is used to reject objects which are older" is, or "is used to accept or reject objects based on their age relative to what is stored" is.